src/components/customerCases/customerCase/CustomerCase.tsx

```diff
@@ -67,9 +67,16 @@ export default function CustomerCase({ customerCase }: CustomerCaseProps) {
             <div key={section._key}>
               {section._type === "richTextBlock" ? (
                 <RichText value={section.richText} />
+              ) : section._type === "quoteBlock" ? (
+                section.quote && (
+                  <div>
+                    <Text>{section.quote}</Text>
+                    {section.author && <Text>- {section.author}</Text>}
+                  </div>
+                )
               ) : (
                 <div className={styles.imageBlockWrapper}>
-                  {section.images.map((image) => (
+                  {section.images?.map((image) => (
                     <div
                       key={image._key}
                       className={styles.imageBlockImageWrapper}
```

Comment:
This feels like it should eventually be pulled out into a switch statement instead of keeping the nested ternary going